Node/vue-browser/src/shims-vue.d.ts

```diff
@@ -0,0 +1,6 @@
+/* eslint-disable */
+declare module '*.vue' {
+  import type { DefineComponent } from 'vue'
+  const component: DefineComponent<{}, {}, any>
+  export default component
+}
```

Comment:
What is this used for? We should comment this block as it may be unclear to see what it is doing

Reply:
This is required, else ts is not able to recognis and import the .vue file.
For reference -https://stackoverflow.com/questions/54622621/what-does-the-shims-tsx-d-ts-file-do-in-a-vue-typescript-project